ozpricechecker/pricefinderapp/tests/models/test_product.py

```diff
@@ -18,6 +20,20 @@ def test_str_no_name(self):
     def test_str_name(self):
         product = Product.objects.create(store=self.store, prod_url='/1234/Tomatoes', name='Super Tomatoes')
         self.assertEqual(str(product), 'Woolworths - Super Tomatoes')
+    
+    def test_latest_prod_price_date(self):
+        product = Product.objects.create(store=self.store, prod_url='/1234/Tomatoes', name='Super Tomatoes')
+        productPrice1 = ProductPrice.objects.create(product=product, price=10)
+        time.sleep(1)
+        productPrice2 = ProductPrice.objects.create(product=product, price=50)
+        time.sleep(1)
+        productPrice3 = ProductPrice.objects.create(product=product, price=25)
+        self.assertEqual(product.latest_price, productPrice3.price)
+        self.assertEqual(product.date_for_latest_price, productPrice3.date_time)
+
+        productPrice3.delete()
+        self.assertEqual(product.latest_price, productPrice2.price)
+        self.assertEqual(product.date_for_latest_price, productPrice2.date_time)
```

Comment:
Can we add a test where the latest date for a product doesn't have a price. Since we are currently still supporting this (and might depending on how we decide) we should still make sure we retrieve the latest price.

Reply:
Not needed since we will wont allow this to happen